src/styles.css

```diff
@@ -0,0 +1,98 @@
+body
+{
+  background-image: url(./img/woman_with_virtual_space_03.jpg);
+  background-repeat: no-repeat;
+  background-size: cover;
+}
+.Header
```

Comment:
@parvkhanna can we use`lower-kebab-case` for class name like `header-container` ?